src/Template/Element/meta.twig

```diff
@@ -3,8 +3,6 @@
 <meta name="author" content="">
 
 <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
```

Comment:
Not even this icon exists in `webroot`, nobody noticed the error because it is used only by apple mobile devices. I would remove this line too.